src/modules/RF69/RF69.h

```diff
@@ -874,10 +874,18 @@ class RF69: public PhysicalLayer {
 #endif
     Module* _mod;
 
+    /*!
+      \brief Proxy to _mod->SPIsetRegValue (to avoid exposing _mod)
+
+      \returns \ref status code
+    */
+    int16_t SPIsetRegValue(uint8_t reg, uint8_t value, uint8_t msb = 7, uint8_t lsb = 0, uint8_t checkInterval = 2);
```

Comment:
Is this still needed with `RADIOLIB_LOW_LEVEL` enabled now?

Reply:
Removed

Reply:
Removed